### ISSUE TITLE
Material select not change focus when press tab-key

### ISSUE THREAD

**Issue by cvillarraga** [InfomediaLtd/angular2-materialize#290]
Hi,

We found a little issue in material-select input, when you click on one option and press the tab-key for go to the next input... not works just re-open the same select options and if choose a diferent option the problem repet again, you need choice the same option twice to can go focus on the next input, however this problem isn't present in materializecss do you know how we can fix this problem?

![download](https://user-images.githubusercontent.com/25514112/27102728-ede780a2-504b-11e7-973e-e878dc0d0d06.png)

**Comment by EverNavarro** [comment]
i've got the same issue

**Comment by rubyboy** [comment]
@cvillarraga I'd like to reproduce this, but so far it seems the bahviour I'm seeing in http://angular2-materialize.surge.sh/#/Forms is the same as the one on the MaterializeCSS website (http://materializecss.com/forms.html).
What I'm seeing is that single select works (moves to the next dropdown), but multi has a problem of needing two tab clicks to move to the next one. 
Is that what you're seeing as well?

**Comment by EverNavarro** [comment]
@rubyboy  Yeah that´s my point, in a multi selector you need to select the option twices for change focus with Tab-Key

**Comment by rubyboy** [comment]
@EverNavarro you're right. I can see the faulty behaviour now. Will try to check if I can figure it out. Thanks.

**Comment by bertvh** [comment]
Any news on this? We're seeing the same issue in our apps. Thanks.

**Comment by harish-rohokale-globant** [comment]
@rubyboy Any updates on this issue?

**Comment by cikal** [comment]
@cvillarraga Did you resolve this issue? or somebody have small tricky about how to set focus in select element? i have same issue here, i try to validate form. when no selected value then i want set focus to select element but no clue how to do.
